applications/luci-app-email/htdocs/luci-static/resources/view/email/emailrelay.js

```diff
@@ -0,0 +1,218 @@
+			_('Path to TLS private key PEM file'));
+		o.datatype = 'file';
+		o.optional = true;
+		o.rmempty = true;
+		o.modalonly = true;
+		o.depends('client_tls', '1');
+
+		o = s.taboption('smtp_client', form.Value, 'client_tls_certificate', _('TLS certificate'),
+			_('Path to TLS cert PEM file'));
+		o.datatype = 'file';
+		o.optional = true;
+		o.rmempty = false;
+		o.modalonly = true;
+		o.depends('client_tls', '1');
+
+		o = s.taboption('smtp_client', form.Value, 'client_tls_verify', _('CA certificate'),
+			_('Verify the remote SMTP server\'s certificate against any of the trusted CA certificates ' +
+				'in the specified file or directory. ' +
+				'The <code>&lt;default&gt;</code> means <code>/etc/ssl/certs/ca-certificates.crt</code>')
+		);
+		o.datatype = 'or(file, directory)';
+		o.default = '<default>';
+		o.optional = true;
+		o.rmempty = false;
+		o.modalonly = true;
+		o.depends('client_tls', '1');
+
+
+		o = s.taboption('pop_server', form.Flag, 'pop', _('Enable POP3'),
+			_('The POP3 used to fetch a mail'));
+		o.default = '0';
+		o.optional = true;
+		o.rmempty = false;
+		o.modalonly = true;
+		o.depends('mode', 'server');
+
+		o = s.taboption('pop_server', form.Value, 'pop_auth', _('POP Auth file'),
+			_('A file containing POP accounts and their credentials. See --pop-auth'));
+		o.datatype = 'file';
+		o.default = '/etc/emailrelay.auth';
+		o.optional = true;
+		o.rmempty = false;
+		o.modalonly = true;
+		o.depends('pop', '1');
+
+		o = s.taboption('pop_server', form.Flag, 'pop_by_name', _('POP by name'),
+			_('Modifies the spool directory used by the POP server to be a sub-directory with the same name as the POP authentication user-id'));
+		o.default = '0';
+		o.optional = true;
+		o.rmempty = false;
+		o.modalonly = true;
+		o.depends('pop', '1');
+
+
+		o = s.taboption('advanced', form.Value, 'spool_dir', _('Mail storage directory'),
+			_('Specifies the directory used for holding mail messages that have been received but not yet forwarded. See --spool-dir'));
+		o.datatype = 'directory';
+		o.default = '/var/spool/emailrelay';
+		o.optional = true;
+		o.rmempty = false;
+		o.modalonly = true;
+
+		o = s.taboption('advanced', form.Value, 'address_verifier', _('Address verifier'),
+			_('Runs the specified external program to verify a message recipient\'s e-mail address. See --address-verifier'));
+		// o.default = "allow:/etc/emailrelay.auth";
+		o.optional = true;
+		o.rmempty = false;
+		o.modalonly = true;
+
+		o = s.taboption('advanced', form.Value, 'filter', _('Filter'),
+			_('Runs the specified external filter program whenever a mail message is stored.'));
+		o.optional = true;
+		o.rmempty = false;
+		o.modalonly = true;
+
+		o = s.taboption('advanced', form.Value, 'extra_cmdline', _('Extra command line options'),
+			_('Specify all arguments that should be passed to EmailRelay. ' +
+				'See <a href=\'http://emailrelay.sourceforge.net/#reference_md_Reference\'>command line reference</a>'));
```

Comment:
Please split the translation in two after '.'
And please fix the alignment oft translation.